packages/core/src/index.ts

```diff
@@ -20,6 +20,7 @@ export type ZodI18nMapOption = {
   t?: i18n["t"];
   ns?: string | readonly string[];
   handlePath?: HandlePathOption;
+  customErrorNs?: string;
```

Comment:
There is no need to separate namespaces.
Multiple namespaces can be specified in `ns` as an array. `ns: ['zod', 'customErrorNS']`.